openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultIntrospectionResultAssembler.java

```diff
@@ -174,8 +174,23 @@ public void shouldAssembleExpectedResultForAccessTokenWithoutExpiry() {
 				.put("token_type", "Bearer")
 				.build();
 		assertThat(result, is(equalTo(expected)));
+		assertFalse(throwException());
+	}
+	
+	private void assertFalse(boolean throwException) {
+		
 	}
 
+	private boolean throwException() throws ParseException{
```

Comment:
You shouldn't need to throw your own exceptions, you're trying to see if the underlying code is going to throw an exception in the right circumstances.